metricbeat/module/apache/apache.asciidoc

```diff
@@ -15,5 +15,21 @@ The Apache Module has currently one MetricSet which is called status. It fetches
 === Apache Status Metricset
 
 
-TODO
-* List available selectors
+The apache status metricset offers the following selectors:
+
+* uptime
+* cpu
+* connections
+* load
+* scoreboard
```

Comment:
Can you add a short description of what these mean.

On line 13, second sentence, it mentions MySQL. I think it's a copy/paste issue.